containers/ddev-webserver/Dockerfile

```diff
@@ -55,6 +55,9 @@ RUN curl --fail -sSL https://raw.githubusercontent.com/netz98/n98-magerun/develo
 RUN curl --fail -sSL https://files.magerun.net/n98-magerun2-latest.phar -o /usr/local/bin/magerun2 && chmod 777 /usr/local/bin/magerun2
 RUN curl --fail -sSL https://raw.githubusercontent.com/netz98/n98-magerun2/develop/res/autocompletion/bash/n98-magerun2.phar.bash -o /etc/bash_completion.d/n98-magerun2.phar && chmod +x /usr/local/bin/magerun
 
+# n for nodejs_version
+RUN npm install -g n
+
```

Comment:
I think it would be better to install `n` directly and then install nodejs using it. That way we don't have the /usr/bin/node and /usr/bin/npm hanging around and causing possible confusion.